src/sql/src/parsers/create_parser.rs

```diff
@@ -144,6 +137,35 @@ impl<'a> ParserContext<'a> {
         Ok(Statement::CreateTable(create_table))
     }
 
+    fn parse_if_not_exist(&mut self) -> Result<bool> {
+        match self.parser.peek_token().token {
+            Token::Word(w) if Keyword::IF != w.keyword => return Ok(false),
+            _ => {}
+        }
+
+        if self.parser.parse_keywords(&[Keyword::IF, Keyword::NOT]) {
+            return self
+                .parser
+                .expect_keyword(Keyword::EXISTS)
+                .map(|_| true)
+                .context(UnexpectedSnafu {
+                    sql: self.sql,
+                    expected: "EXISTS",
+                    actual: self.peek_token_as_string(),
+                });
+        }
+
+        if self.parser.parse_keywords(&[Keyword::IF, Keyword::EXISTS]) {
+            return UnsupportedSnafu {
+                sql: self.sql,
```

Comment:
non-blocking: Maybe we should remove the sql from the error message.

Reply:
IIRC this error goes to client and we don't log it. So it should be fine to keep as is.

We may investigate this manner later and not couple it with this patch :D It's anyway not a regression.